### PR TITLE
Create Chef's Masterpieces website

### PR DESCRIPTION
**Pull request by FuadNabi**
PullRequestGIF
# In This Pull Request

- Created a Home page and details page
-  Implemented styles for the home page and details page
-  Implemented a Responsive design
-  Used fake store APIs
-  Fetch data from faker store APIS using Axios
-  Used Redux toolkit to manage state across the web app
-  Implemented slice for product and products
-  Used Create Async Thunk method provided by the redux toolkit to handle the API requests
-  Implemented Filter Feature for categories
-  Implemented Unit testing for components and redux pure functions

## Link to Live version and presentation
https://63bea92097774011416d56b1--lucky-baklava-595536.netlify.app/ (Live)
https://www.loom.com/share/835c81073f77465a8cb76146700bc73e (Presentation)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,164 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+Give a **Star** if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```
Comment (mckent05) [inline README.md:160]: - Please include an acknowledgment section giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @FuadNabi, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your project looks good 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (FuadNabi) [reply README.md:160]: Thanks for review

Comment (Abenezer-Tilahun) [reply README.md:160]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @FuadNabi 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**